docs/additional/interesting_aspects_de.md

```diff
@@ -0,0 +1,248 @@
+Wir empfehlen, dass eine lokale Entwicklungsumgebung in Form einer `docker-compose.yml`-Datei definiert ist und jederzeit einfach mit `docker-compose up` gestartet und mit `docker-compose down` heruntergefahren werden kann.  
+Für ein Beispiel [siehe CAS][local-cas-example].
+
+[local-cas-example]: https://github.com/cloudogu/cas/blob/develop/app/docker-compose.yml
+
+## Qualitätssicherung für Dogus
+
+### Container Validation (GOSS)
+
+Die Container-Validation versichert die richtige Konfiguration des Containers, die für eine reibungslose Ausführung der Software notwendig ist.  
+
+Dafür verwenden wir in unseren Dogus das Server-Validation Programm [goss][goss].  
+Die eigentlichen goss-Tests werden in der folgenden Datenstruktur im Dogu angelegt:
+- `<Root-Verzeichnis des Dogus>/`
+  - `spec/`
+    - `goss/`
+      - `goss.yml` (enthält alle goss-Tests)
+
+Die goss-Tests sollten am besten alle wichtigen Aspekte für den reibungslosen Start eines Dogus überprüfen.  
+Darunter fallen Aspekte wie:
+- Überprüfen der Berechtigung für Skripte (startup.sh), Volumes
+- Überprüfen der UID & GID für Dateien (startup.sh, resources, Volumes, Schreib-abhängige Pfade)
+- Erreichbarkeit der Software: TCP-Health-Checks
+
+Für die Integration in eine Pipeline empfehlen wir den Einsatz der `ecoSystem.verify()` Methode aus der [dogu-build-lib][dogu-build-lib].  
+Ein beispielhafter Aufruf sieht in einem Jenkinsfile wie folgt aus:
+
+```groovy
+stage('Verify') {
+    ecoSystem.verify("/dogu")
+}
+```
+
+Dieser ruft intern den Befehl `cesapp verify` auf, welcher die Goss-Tests ausführt.
+
+[goss]: https://github.com/goss-org/goss
+[dogu-build-lib]: https://github.com/cloudogu/dogu-build-lib
+
+## Dokumentation
+
+Jedes Tool sollte eine sinnvolle Dokumentation über verwendete Features besitzen, die während der Benutzung oder Support unterstützt.
+Um eine einfache Integration in unsere vorhandene Dokumentations-Infrastruktur zu ermöglichen, sollte die Dokumentation in einem im Repository-Root befindlichen Order `/docs` abgelegt werden.
+
+Standardmäßig umfasst die Doku folgende Dinge:
+- Features von dem Dogu
+- Installation des Dogus
+- Konfiguration des Dogus
+- Entwicklung am Dogu
+- Readme
+- Changelog
+
+### Lizenz
+
+Die Nutzungslizenz des Dogus sollte deutlich hervorgehen.
+Diese kann zum Beispiel in der Readme angegeben werden.
+
+### Readme
+
+Eine Readme ist sowohl für eigene Entwicklungen als auch im Partnerkontext mit externen Entwicklern sinnvoll.  
+Bei öffentlichen Repositories sollte jedoch ein höherer Maßstab an einer Readme angelegt werden.
+
+Eine Readme-Datei liegt im Repository-Root und beschreibt knapp Hinweise zum generellen Zweck des Dogus.
+Dies kann in Form eines Quickstart-Guides sein, indem zum Beispiel beschrieben ist wie man eine lokale Entwicklungsumgebung einrichtet.
+Es sollte außerdem ein Verweis auf den Dokumentationsordner beinhalten und kann darüber hinaus auch auf spezielle Seiten der Dokumentation verweisen.
```

Comment:
This sentences mixes two things which should be separated:
1. describing the purpose of the dogu (aimed at dogu users)
2. a development quickstart (aimed at dogu developers)

I suggest to make the possible uses of a Readme more plastic, like so:

```suggestion
Eine Readme-Datei liegt im Repository-Root und beschreibt knapp Hinweise zum generellen Zweck des Dogus. Für Anwender kann dies in Form eines Quickstart-Guides sein, wie man das Dogu verwendet oder Auflistungen von Features. Für Dogu-Entwickler könnte die Readme Hinweise auf Kollaboration eingehen oder auf die verwendete Urheberlizenz hinweisen. Verlinkung auf weitere Dokumentation stellt zudem ein Mittel dar, um sowohl Übersicht zu wahren als auch weitere Informationen anzubieten.
```